teku/src/test/java/tech/pegasys/teku/cli/options/ValidatorOptionsTest.java

```diff
@@ -154,6 +154,15 @@ public void shouldEnableValidatorRegistrationTimestampOverride() {
         .isEqualTo(Optional.of(UInt64.valueOf(120000)));
   }
 
+  @Test
+  public void shouldEnableValidatorRegistrationPubkeyOverride() {
+    final String[] args = {"--validators-builder-registration-pubkey-override", "120000"};
```

Comment:
120000 needs to be changed to random BLSPublicKey 

not sure if there's a function for random 32 bytes